pinot-plugins/pom.xml

```diff
@@ -90,7 +90,19 @@
         <plugins>
           <plugin>
             <artifactId>maven-shade-plugin</artifactId>
-            <version>3.2.1</version>
+            <version>3.3.0</version>
```

Comment:
reason for upgrading this?

Reply:
cause this bouncycastle lib has multi-release, so the shade plugin doesn't have a compilation error. The fix is to add dependency of asm for this. I can revert the version upgrade and retry.